articles/iot-accelerators/iot-accelerators-connecting-pi-node.md

```diff
@@ -82,7 +82,6 @@ Complete the following steps using the `ssh` connection to your Raspberry Pi:
 1. To download and install the packages you need to complete the sample app, run the following commands:
 
     ```sh
-    npm init
```

Comment:
Although not strictly necessary - it's useful to run `npm init` so that when you run the next line, the package versions are saved into package.json.

If you do remove `npm init`, you should remove `--save` on the next line as it's redundant.

Reply:
Sounds good will remove the --save